src/libos/src/process/do_wait4.rs

```diff
@@ -30,6 +31,9 @@ pub fn idle_reap_zombie_children() -> Result<()> {
         }
     }
 
+    // Also check for vforked child process
+    reap_zombie_child_created_with_vfork();
```

Comment:
I don't think this function should be called here, this function should be used to clean up memory when libos is shutdown.

Reply:
Yes. This is wrong. I re-implement this function and call it in `exit_process`. Please take a new look. Thank you for your review.